README.md

```diff
@@ -15,18 +15,14 @@ Yoti SDK carries out for you steps 6, 7 ,8 and the profile decryption in step 9.
 
 Yoti also allows you to enable user details verification from your mobile app by means of the Android (TBA) and iOS (TBA) SDKs. In that scenario, your Yoti-enabled mobile app is playing both the role of the browser and the Yoti app. By the way, your back-end doesn't need to handle these cases in a significantly different way. You might just decide to handle the `User-Agent` header in order to provide different responses for web and mobile clients.
 
+## Prerequisites
+Check you have all the following before starting your integration.
+Please ensure you have created your application in our YOTI dashboard and have downloaded/made a note of all the keys. Keep them safe as you'll need them soon!
 
-## Enabling the SDK
-To import the Yoti SDK inside your project, you can use your favourite dependency management system.
-If you are using NPM, you need to add the following dependency:
-
-```json
-"dependencies": {
-    "yoti-node-sdk" : "1.0.0"
-}
-```
+## Supported Versions
+The Yoti Nodejs SDK currently supports all versions for Mac users. For Window users please ensure you have version 7.8.0 or later.
```

Comment:
A few things:

1) You are not specifically saying Node version. Ergo saying 'all versions for Mac' it could be misinterpreted as Mac versions like El Capitan or Sierra or whatever.
2) I see nothing in the SDK that prevents users on Windows using < 7.8.0. I would not put this in here.
3) You are ignoring Linux users

@xchangebit Are there actually any limitations to Node versions on any platform?